build.gradle

```diff
@@ -18,13 +18,15 @@ compileGroovy {
     targetCompatibility = '1.6'
 }
 
+def ApacheHttpComponentsVersion = '4.5.2'
+
 // Build dependencies
 dependencies {
     compile gradleApi()
     compile localGroovy()
-    compile 'com.android.tools.build:gradle:1.5.0'
-    compile 'org.apache.httpcomponents:httpclient:4.3.6'
-    compile 'org.apache.httpcomponents:httpmime:4.3.6'
```

Comment:
Per the previous discussion about the dependency versioning, I’d rather introduce new versions of httpclient separately from this feature, as it breaks the configuration of existing users. The plugin also supports gradle 1.5.0+, so I don’t see a reason to change the lower bound.